utils/zerovec/README.md

```diff
@@ -0,0 +1,44 @@
+# zerovec [![crates.io](http://meritbadge.herokuapp.com/zerovec)](https://crates.io/crates/zerovec)
+
+Zero-copy vector abstractions over byte arrays.
+
+This crate has two main types:
+
+- `ZeroVec<T>` for fixed-width types like `u32`
+- `VarZeroVec<T>` for variable-width types like `str`
+
+Both are intended as drop-in replacements for `Vec<T>` in Serde structs serialized with a
+format supporting a borrowed byte buffer, like Bincode.
```

Comment:
`request`: Can you add a bit more about value proposition here? Why something new, what makes `Vec` suboptimal, what `ZeroVec` brings to the table?

I understand that first line says that this is zero-copy, which may be interpreted by the reader as "ah, so Vec must not be zero copy", but I think it's useful to add something like:

> This utility can be useful in scenarios such as X, where `Vec` would cause Y, while `ZeroVec` allows for Z.

Reply:
Done as requested, in lib.rs and the auto-generated README.md.